orttraining/orttraining/python/training/pytorch_trainer_options.py

```diff
@@ -0,0 +1,417 @@
+import cerberus
+import torch
+
+from .optim import lr_scheduler
+from .amp import loss_scaler
+
+
+class PytorchTrainerOptions(object):
```

Comment:
Was this class suppposed to be named ORTTrainerOptions?

Reply:
Yes, I have renamed it (temporarily?) to prevent confusion with the existing `ORTTrainer` class. I also intend not to replace the `ORTTrainer` class and instead create a `PytorchTrainer` or `ONNXRuntimeTrrainer` class until we finish the new frontend and be able to delete `ORTTrainer` completely. Having both working will be useful for comparing numbers, performance and capabilities. Is it ok?